src/app/plugins/delclasschans.plugin.ts

```diff
@@ -0,0 +1,34 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType, ClassType } from '../../common/types';
+
+export class DeleteClassChannelsPlugin extends Plugin {
+  public name: string = ' Delete classes';
+  public description: string = "Nukes every class channel. Don't be idiot";
+  public usage: string = 'delclasschans [super secret password]';
+  public permission: ChannelType = ChannelType.Admin;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public validate(message: IMessage, args: string[]) {
+    return (
+      args.join(' ') ===
+      'Yes I understand that this is a really really serious thing to do and I will not be stupid and do it prematurely or maliciously.'
```

Comment:
Still debating if it's even worth to include this ...

Reply:
let's not, but also just not deploy this patch till it's time to delete, and then we can disable the plugin till next end of sem